src/displayBackend/wayland/Display.cpp

```diff
@@ -498,7 +501,10 @@ void Display::init()
 		throw Exception("Can't get registry", errno);
 	}
 
-	wl_registry_add_listener(mWlRegistry, &mWlRegistryListener, this);
+	if(-1 == wl_registry_add_listener(mWlRegistry, &mWlRegistryListener, this))
+	{
+		throw Exception("Can not add the listener.", errno);
```

Comment:
This change is not described in the commit message.

Also, AFAIR, we don't use `if ( const == f() )` style.

Reply:
replaced

Reply:
Commit should do one atomic change. In this case you clearly saying "that does not have a relation to the issue", so it is better to do in the separate commit. You can create that commit in this PR.